client/dfclient/src/fov.cpp

```diff
@@ -0,0 +1,383 @@
+#include <cassert>
+#include <cmath>
+#include <set>
+#include <queue>
+#include <functional>
+
+#include "fov.hpp"
+
+namespace fov {
+
+static const bool doDeoverlap = true;
```

Comment:
I don't really see a case when we would need a not deoverlapped shape, I think we can remove this

Reply:
Removed